### DISCUSSION TITLE
Template Specs  or Powershell-Script Deployment

### DISCUSSION THREAD

**Discussion by iflach** [Azure/AzOps#670] [Q&A]
Hi all, 
is there a way to deploy Azure Template Specs with linked ARM templates (with relative path ) or run a Powershell script that deploys the template specs ?

Thanks in advance!

**Comment by StefanIvemo** [comment]
You could use the template spec as a module in a bicep template to get it deployed using AzOps. 

https://docs.microsoft.com/en-us/azure/azure-resource-manager/bicep/modules#file-in-template-spec

**Reply by iflach** [reply]
The problem is , we have an older Version from AzOps so we only can Deploy ARM-Templates.

**Reply by daltondhcp** [reply]
Is there anything blocking you from moving to the latest version of AzOps?

**Reply by iflach** [reply]
FYI : I solved the problem with this ARM-Template : https://docs.microsoft.com/en-us/azure/templates/microsoft.resources/templatespecs/versions?tabs=json

**Reply by StefanIvemo** [reply]
Aha, you wanted to create new template specs. I thought you wanted to deploy various resources using existing template specs.

**Reply by iflach** [reply]
Ok , it is confusing that you have to deploy it 2x. Once to Azure and once from Azure.
 I wanted to create(deploy) template specs with LinkedTemplates and deploy them afterwards with policies when the appropriate tags are set.
 
 But anyway , thanks for the Quick Replies  👍
